src/base.ts

```diff
@@ -873,4 +873,13 @@ export default class KintoClientBase {
       )
     );
   }
+
+  /**
+   * Retrieves server endpoints.
+   *
+   * @return {Object}
+   */
+  get endpoint(): Record<string, Function> {
```

Comment:
can you change the return type to `typeof endpoint`? Technically `Function` is a wider type than what we're using.